android/README.md

```diff
@@ -0,0 +1,443 @@
+To get information, you can use the `TokenProvider`. This will assist you in getting token-related information and performing actions on the tokens.
+
+```kotlin
+val tokenProvider: TokenProvider = asgardeoAuth.getTokenProvider()
+```
+
+To get the token-related information, you can use the following functions:
+
+```kotlin
+val accessToken: String? = tokenProvider.getAccessToken(context)
+val idToken: String? = tokenProvider.getIDToken(context)
+val refreshToken: String? = tokenProvider.getRefreshToken(context)
+val accessTokenExpirationTime: Long? = tokenProvider.getAccessTokenExpirationTime(context)
+val scope:String? = tokenProvider.getScope(context)
+```
+
+### Perform action based on the tokens
+
+If you want to perform any action based on the tokens that are returned, you can use the `performAction` function in the `TokenProvider`.
+
+```kotlin
+tokenProvider.performAction(context) { accessToken, idToken, ->
+    action(accessToken, idToken)
+}
+```
+
+### Logout
+
+If you want to perform a logout, you can call the `logout` function in the `AuthenticationProvider`. This will emit the state `AuthenticationState.Initial` if the logout is successful, and if an error occurs, it will emit `AuthenticationState.Error`.
+
+```kotlin
+authenticationProvider.logout(context)
+```
+
+### Client attestation
+You can also use client attestation with the SDK as well.
+
+#### How to setup client attestation for your application
+1. In the created mobile application in the Asgardeo, go to the advanced section and enable the client attestation.
+2. You need to subscribe to the "Google Play Integrity API" from your Google cloud project, you can do this from Enabled API's & Services in your Google cloud project.
+3. After that you need to create a new service account in the google project. You can create a Service Account for yourself with the following steps.
+
+    1. Go to `IAM & Admin` -> `Service Accounts`.
+    2. Click Create Service Account.
+    3. Fill the name and click create and continue.
+    4. You need to grant your service account the roles of Service Account User and Service Usage Consumer.
+    5. Click continue and then Done
+    6. You can see the service account added without keys, click : Actions -> Manage Keys for the service account.
+    7. Click Add key and Select JSON.
+    8. Save the JSON in secure place (We need this for Android Attestation Credentials for application metadata)
+
+4. After that, Update Application Advanced properties. The application you created requires 2 properties to perform android attestation.
+
+    1. Android package name
+    2. androidAttestationServiceCredentials
+       The JSON secret of Service Account downloaded. Note that this attribute is defined as a JSON object hence use the JSON key as it is.
+
+5. Now you can call the "Google Play Integrity API" from your application and pass the integrity token that you will get from the API result to the integrityToken value of the AuthenticationCoreConfig
```

Comment:
Shall we move this info to Asgardeo docs and add a link to docs here?

Reply:
there is an ongoing effort for this by another team, to bring docs related to app native authentication to Asgardeo docs.